src/components/WebexLocalMedia/__snapshots__/WebexLocalMedia.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Webex Local Media component snapshot matches snapshot of disabled local video 1`] = `
 <div
-  className="wxc-local-media wxc-no-media"
+  className="wxc-local-media no-media"
```

Comment:
Wouldn't we want `no-media` to also be prefixed? it seems like this could clash with other CSS class names